modules/widgets/facebook-likebox.php

```diff
@@ -105,7 +105,7 @@ function widget( $args, $instance ) {
 
 		?>
 		<div id="fb-root"></div>
-		<div class="fb-page" data-href="<?php echo esc_url( $page_url ); ?>" data-width="<?php echo intval( $like_args['width'] ); ?>"  data-height="<?php echo intval( $like_args['height'] ); ?>" data-hide-cover="<?php echo esc_attr( $like_args['cover'] ); ?>" data-show-facepile="<?php echo esc_attr( $like_args['show_faces'] ); ?>" data-show-posts="<?php echo esc_attr( $like_args['stream'] ); ?>">
+		<div class="fb-page" data-href="<?php echo esc_url( $page_url ); ?>" <?php echo ( $like_args['width'] == 0 ) ? 'data-adapt-container-width="true"' : 'data-adapt-container-width="false" data-width="'. intval( $like_args['width'] ) . '"'; ?> data-height="<?php echo intval( $like_args['height'] ); ?>" data-hide-cover="<?php echo esc_attr( $like_args['cover'] ); ?>" data-show-facepile="<?php echo esc_attr( $like_args['show_faces'] ); ?>" data-show-posts="<?php echo esc_attr( $like_args['stream'] ); ?>">
```

Comment:
What happens if we do not add the parameter at all as default?

```suggestion
		<div class="fb-page" data-href="<?php echo esc_url( $page_url ); ?>" <?php echo ( 0 === $like_args['width'] ) ? 'data-adapt-container-width="true"' : ''; ?> data-width="'. intval( $like_args['width'] ) . '"'; ?> data-height="<?php echo intval( $like_args['height'] ); ?>" data-hide-cover="<?php echo esc_attr( $like_args['cover'] ); ?>" data-show-facepile="<?php echo esc_attr( $like_args['show_faces'] ); ?>" data-show-posts="<?php echo esc_attr( $like_args['stream'] ); ?>">
```

In the above, I also switched to a strict comparison, and switched to a Yoda condition.